README.md

```diff
@@ -352,7 +352,7 @@ end
 
 See the [Datadog integration Chef recipes][7] for additional examples.
 
-### datadog_integration
+### Datadog integration
```

Comment:
Same here, `datadog_integration` is a thing our cookbook defines. Alternative title: "Install or upgrade a specific version of an integration"